chart/openfaas/templates/oauth2-plugin-dep.yaml

```diff
@@ -0,0 +1,121 @@
+{{- $functionNs := default .Release.Namespace .Values.functionNamespace }}
+{{- if .Values.oauth2Plugin }}
```

Comment:
I dont think this line is needed because the default values file will make sure this is not an empty object

Reply:
Yes.. I have added it to all the values files, you're right,.